src/com/company/zadania/obiektowe/zadanie1/Pozycja.java

```diff
@@ -27,6 +27,18 @@ public double obliczWartosc() {
         return cena * ileSztuk;
     }
 
+    public String getNazwaTowaru() {
+        return nazwaTowaru;
+    }
+
+    public void setIleSztuk(int ileSztuk) {
+        this.ileSztuk += ileSztuk;
```

Comment:
setIleSztuk powinno ustawiac konkretna liczbe sztuk czyli this.ileSztuk =  ileSztuk
a nie addDodatkoweIleSztuk() -> this.ilesztuk +=..